### PR TITLE
Style guide addition to Product, capability page

### PR DESCRIPTION
**Pull request by bhecknr**
Revised the trademarks info, which was reviewed by various members of the style committee and approved by Legal.

### CODE REVIEW

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 1]
```diff
@@ -15,17 +15,33 @@ Do not use acronyms specific to or coined by New Relic externally; only use indu
   Find capitalization guidelines for [user types](/docs/style-guide/word-choice/user-related-language-guidelines#styles) and [product editions](/docs/style-guide/word-choice/pricing-language-guidelines#style-format).
 </Callout>
 
+## Trademarks defined
+
+A trademark is a word, name, symbol, or a combination of the three that identifies a particular product or service. [It's how customers recognize you in the marketplace and distinguish you from your competitors for those goods and services] (https://www.uspto.gov/trademarks/basics/what-trademark#:~:text=A%20trademark%20can%20be%20any,both%20trademarks%20and%20service%20marks).
+
+A trademark name is the name of a company conducting business. The New Relic trade name is New Relic, Inc.
```
Comment (mmfred) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:22]:
```suggestion
A trademark name is the name of a company conducting business. The New Relic trademark name is New Relic, Inc.
```

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 2]
```diff
@@ -15,17 +15,33 @@ Do not use acronyms specific to or coined by New Relic externally; only use indu
...
+
+It’s not necessary to use trademark characters after product names in headings and text on newrelic.com webpages.
+
+## Third-party trademarks
+
+In general, it’s not necessary or recommended to use trademark characters after third-party product names on newrelic.com webpages or documents (including PDFs). New Relic is typically not in the position to know if a particular mark is registered. Assuming a third-party product name or other third-party mark is registered can lead to unintended and undesired consequences for New Relic. Thus, avoid using the ® or ™ symbol unless a third party that New Relic partners or works with specifically requires the use of the symbol as a part of its trademark policy.
+
+In some instances, a third party requires the use of the ® or ™ symbol by contract or through its Trademark Policy. For example, LEGO. A recent FutureStack event promoted an activity in which a LEGO master would create a LEGO model. In this case, LEGO required that the first occurrence of LEGO in New Relic copy should be followed by ®. For example, "You can win a one-of-a-kind LEGO® model of the New Relic Data Nerd Knuckles."
```
Comment (mmfred) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:42]:
```suggestion
In some instances, a third party requires the use of the ® or ™ symbol by contract or through its trademark policy. For example, LEGO. A recent FutureStack event promoted an activity in which a LEGO master would create a LEGO model. In this case, LEGO required that the first occurrence of LEGO in New Relic copy should be followed by ®. For example, "You can win a one-of-a-kind LEGO® model of the New Relic Data Nerd Knuckles."
```

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 3]
```diff
@@ -15,17 +15,33 @@ Do not use acronyms specific to or coined by New Relic externally; only use indu
...
+
+If a company we are working with has its trademark policy publicly available or has privately provided the policy to New Relic, please consult that policy and follow the company’s policy for use of the ® or ™ symbol.  If you are unsure of whether you need to acknowledge a third-party trademark in your copy, contact Legal in the #suplegal channel in Slack.
```
Comment (mmfred) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:44]:
```suggestion
If a company we're working with has its trademark policy publicly available or has privately provided the policy to New Relic, please consult that policy and follow the company’s policy for use of the ® or ™ symbol.  If you're unsure of whether you need to acknowledge a third-party trademark in your copy, contact Legal in the #suplegal channel in Slack.
```

**Review by mmfred — COMMENTED** [review]
I made some minor copy edits.

**Comment by bhecknr** [comment]
Good catches. Thanks.

**Pull request merged**